themes/powerline-multiline/README.md

```diff
@@ -59,6 +59,7 @@ The contents of both prompt sides can be "reordered", all the "segments" (every
 * `clock` - Current time in `HH:MM:SS` format
 * `cwd` - Current working directory including full folder hierarchy (c.f. `wd`)
 * `hostname` - Host name of machine
+* `in_toolbox` - Show identifier if running inside a toolbox
```

Comment:
Maybe link to the toolbox project page so ppl can discover what this is referring to:

```
* `in_toolbox` - Show identifier if running inside a [toolbox](https://github.com/containers/toolbox)
```

(adjust link if a better one exists)

Reply:
Thanks. Good suggestion.